tests/template/logparser_data/shadow1.txt

```diff
@@ -0,0 +1,291 @@
+     2750000    rays have been rejected so far.
+       31362                   accepted.
+     2800000    rays have been rejected so far.
+       31926                   accepted.
+     2850000    rays have been rejected so far.
+       32472                   accepted.
+                  32500
+     2900000    rays have been rejected so far.
+       33071                   accepted.
+     2950000    rays have been rejected so far.
+       33634                   accepted.
+     3000000    rays have been rejected so far.
+       34218                   accepted.
+     3050000    rays have been rejected so far.
+       34745                   accepted.
+                  35000
+     3100000    rays have been rejected so far.
+       35329                   accepted.
+     3150000    rays have been rejected so far.
+       35846                   accepted.
+     3200000    rays have been rejected so far.
+       36420                   accepted.
+     3250000    rays have been rejected so far.
+       36937                   accepted.
+                  37500
+     3300000    rays have been rejected so far.
+       37542                   accepted.
+     3350000    rays have been rejected so far.
+       38083                   accepted.
+     3400000    rays have been rejected so far.
+       38637                   accepted.
+     3450000    rays have been rejected so far.
+       39191                   accepted.
+     3500000    rays have been rejected so far.
+       39748                   accepted.
+                  40000
+     3550000    rays have been rejected so far.
+       40362                   accepted.
+     3600000    rays have been rejected so far.
+       40935                   accepted.
+     3650000    rays have been rejected so far.
+       41494                   accepted.
+     3700000    rays have been rejected so far.
+       42031                   accepted.
+                  42500
+     3750000    rays have been rejected so far.
+       42618                   accepted.
+     3800000    rays have been rejected so far.
+       43185                   accepted.
+     3850000    rays have been rejected so far.
+       43779                   accepted.
+     3900000    rays have been rejected so far.
+       44332                   accepted.
+     3950000    rays have been rejected so far.
+       44895                   accepted.
+                  45000
+     4000000    rays have been rejected so far.
+       45453                   accepted.
+     4050000    rays have been rejected so far.
+       46032                   accepted.
+     4100000    rays have been rejected so far.
+       46596                   accepted.
+     4150000    rays have been rejected so far.
+       47170                   accepted.
+                  47500
+     4200000    rays have been rejected so far.
+       47720                   accepted.
+     4250000    rays have been rejected so far.
+       48313                   accepted.
```

Comment:
above lines @moellep ?

Reply:
The example log files have the full log - you only want to get the error parts out.

Reply:
The full shadow log should result in one error line: "Compound is not a valid chemical formula and is not present in the NIST compound"